sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/CLIServiceClient.java

```diff
@@ -35,7 +36,7 @@ public SessionHandle openSession(String username, String password)
   }
 
   @Override
-  public RowSet fetchResults(OperationHandle opHandle) throws HiveSQLException {
+  public TRowSet fetchResults(OperationHandle opHandle) throws HiveSQLException {
```

Comment:
CLIServiceClient and its child is used in our test only, changes here will not have impacts on real-world clients